usage/usage.py

```diff
@@ -77,9 +72,42 @@ def gather_data(args):
     proc = Popen(argv, stdout=PIPE, stderr=PIPE, encoding='utf-8')
     stdout, stderr = proc.communicate()
     if stderr != '':
-        print(stderr)
-        raise OSError('failed to gather data.')
-
+        # This is to catch the jobs that have conflicting JOB_TERMINATED
+        # records, but only if the job ultimately completes correctly.
+        if "Conflicting JOB_TERMINATED record (COMPLETED)" in stderr:
+            err_num=[]
+            # Finds jobIDs of possibly completed jobs in STDERR
+            for line in stderr.split("\n"):
+                if "Conflicting JOB_TERMINATED record (COMPLETED)" in line:
+                    err = [int(s) for s in line.split() if s.isdigit()]
+                    err_num+=err
+            err_list=[]
+            for i in range(len(err_num)):
+                if i%2==0:
+                    err_list.append(str(err_num[i]))
```

Comment:
Based on example output in the JIRA ticket, the entire logic in lines 75-87 can be expressed more clearly using regular expressions (remember to to put `import re` and the beginning of the module):
```python
if "Conflicting JOB_TERMINATED record (COMPLETED)" in stderr:
    err_list = []
    for line in stderr.split('\n'):
        match = re.search('\d+', line)
        err_list.append(match.group(0))
```
If you decide to keep your version please fix spaces around operators in lines 78, 83, 84, and 86.